src/components/VSelect/mixins/select-watchers.js

```diff
@@ -90,7 +90,9 @@ export default {
       // Wrap input to next line if overflowing
       if (this.$refs.input.scrollWidth > this.$refs.input.clientWidth) {
         this.shouldBreak = true
-        this.$nextTick(this.$refs.menu.updateDimensions)
+        if (this.isActive) {
```

Comment:
Shouldn't `this.isActive` be checked in the nextTick? Same in `selectedItems()`

Reply:
Not sure...